.github/workflows/packages.yml

```diff
@@ -112,6 +122,13 @@ jobs:
         sudo cmake --install build --prefix /usr --config RelWithDebInfo
         popd
 
+    - name: Install qaseprite plugin
+      working-directory: qaseprite
+      run: |
+        cmake -B build -DCMAKE_BUILD_TYPE=Release
```

Comment:
The plugin should be build against the system libraries, as done in https://github.com/mapeditor/qaseprite/blob/c388282e746fa41e15b38750b502f7a8e4361cad/.github/workflows/build.yml#L51-L55 (also needs those dependencies installed).